components/login/login.css

```diff
@@ -0,0 +1,45 @@
+.nav-container{
+    z-index: 3;
+}
+.nav-title{
+    font-size: 1.3rem;
+  }
+  .nav-icon{
+    font-size: 1.3rem;
+  }
+.main-content{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    gap: 1rem;
+    padding: 1.75rem 0;
+}
+ .description{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 2rem;
+} 
+
+.auth-form{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+}
```

Comment:
suggestion: you can make utility classes for flex because you are using it quite often.

Reply:
Sure, I'll keep that in mind from now on